CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Fix [#1643](https://github.com/Glavin001/atom-beautify/issues/1643). Add PHPCBF Version setting and remove "--no-patch" option from version 3.0 of PHPCBF. See [#1645](https://github.com/Glavin001/atom-beautify/pull/1645).
 - Close [#27](https://github.com/Glavin001/atom-beautify/issues/27). Add commands for beautifying a specific language
 - Add "Allow Risky" option for PHP-CS-Fixer. See [#1614](https://github.com/Glavin001/atom-beautify/pull/1614)
+- Fix [#1647](https://github.com/Glavin001/atom-beautify/issues/1647), avoiding a spurious error message in the console.
```

Comment:
This message could be more informative. Consider `Set Yaml padding default value to 0, removing Atom warning message "'atom-beautify.yaml.padding' could not set the default."`.